src/site/content/en/learn/css/pseudo-elements/index.md

```diff
@@ -0,0 +1,301 @@
+You can also insert a
+[`counter`](https://developer.mozilla.org/en-US/docs/Web/CSS/counter()).
+
+{% Aside 'key-term' %}
+You can create a named counter and then increment it,
+based on it's position in the document flow.
+There's all sorts of contexts where they can be really useful.
+{% endAside %}
+
+Once a `::before` or `::after` element has been created,
+you can style it however you want with no limits.
+You can only insert a `::before` or `::after` element to an element that will accept child elements
+([elements with a document tree](https://www.w3.org/TR/CSS21/generate.html)),
+so elements such as `<img />`, `<video>`, `<button`> and `<input>` won't work.
+
+{% Codepen {
+  user: 'web-dot-dev',
+  id: 'GRrEYrg'
+} %}
+
+## `::first-letter`
+
+We met this pseudo-element at the start of the lesson.
+It is worth being aware that not all CSS properties can be used when targeting
+[`::first-letter`](https://developer.mozilla.org/en-US/docs/Web/CSS/::first-letter).
+The available properties are:
+
+- `color`
+- `background` properties (such as `background-image`)
+- `border` properties (such as `border-color`)
+- `float`
+- `font` properties (such as `font-size` and `font-weight`)
+- text properties (such as `text-decoration` and `word-spacing`)
+
+```css
+p::first-letter {
+  color: goldenrod;
+  font-weight: bold;
+}
+```
+
+{% Codepen {
+  user: 'web-dot-dev',
+  id: 'JjEJmOx'
+} %}
+
+{% Aside %}
+You can only use `:first-letter` on block containers.
+Therefore, it won't work if you try to add it to an element that has `display: inline`.
+{% endAside %}
+
+## `::first-line`
+
+The [`::first-line`](https://developer.mozilla.org/en-US/docs/Web/CSS/::first-line)
+pseudo element will let you style the first line of text
+only if the element with `::first-line` applied has a `display` value of `block`,
+`inline-block`, `list-item`, `table-caption` or `table-cell`.
+
+```css
+p::first-line {
+  color: goldenrod;
+  font-weight: bold;
+}
+```
+
+{% Codepen {
+  user: 'web-dot-dev',
+  id: 'vYgZVaO'
+} %}
+
+Like the `::first-letter` pseudo element,
+there's only a subset of CSS properties you can use:
+
+- `color`
+- `background` properties
+- `font` properties
+- `text` properties
+
+## `::backdrop`
+
+If you have an element that is presented in full screen mode,
+such as a `<dialog>` or a `<video>`,
+you can style the backdrop—the space between the element and the rest of the page—with the
+[`::backdrop`](https://developer.mozilla.org/en-US/docs/Web/CSS/::backdrop) pseudo element:
+
+
+```css
+video::backdrop {
+  background-color: goldenrod;
+}
+```
+
+{% Codepen {
+  user: 'web-dot-dev',
+  id: 'rNjwqRQ'
+} %}
+
+The `::backdrop` pseudo element is supported in all major browsers apart from Safari.
+
+## `::marker`
+
+The [`::marker`](https://developer.mozilla.org/en-US/docs/Web/CSS/::marker)
+pseudo element lets you style the bullet or number for a list item or the arrow of a `<summary>` element.
+
+```css
+::marker {
+  color: olive;
```

Comment:
Can we make this example a more vivid color such as `red` or `hotpink` to help it stand out from the black text? Olive is nice but more subtle. I'd update here and in the demo :)

Reply:
Making the update!